src/Configuration.ts

```diff
@@ -92,6 +92,11 @@ export const configurations: { [key: string]: Configuration } = {
         emailDomain: 'nijmegen.nl',
         emailFromDomain: 'amazonses.nijmegen.nl',
       },
+      {
+        name: 'sandbox-01-nijmegen',
```

Comment:
Dit gaat goed, name wordt alleen gebruikt als cdk logicalid en in een error message. Hoeft dus niet overeen te komen met hostezone name.

Reply:
Dat was mijn aanname gezien de benaming van de webform config met -nijmegen erachter. Thnx voor de check